kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/OperatorMainIT.java

```diff
@@ -7,8 +7,10 @@
 package io.kroxylicious.kubernetes.operator;
 
 import org.assertj.core.api.Assumptions;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+@Disabled
```

Comment:
```suggestion
@EnabledIf(value = "io.kroxylicious.kubernetes.operator.OperatorTestUtils#isKubeClientAvailable", disabledReason = "no viable kube client available")
```

Reply:
I've disabled this entirely since I _think_ it conflicts with the other IT, leaving an operator running that messes with following tests. Will add a TODO comment or something.

Reply:
Ahh ok. I'm looking at tearing it down properly in #1867 anyway